qa/SidechainTestFramework/sc_test_framework.py

```diff
@@ -59,68 +61,100 @@ def sc_setup_chain(self):
         initialize_default_sc_chain_clean(self.options.tmpdir, 1)
 
     """
-    Bootstrap some sidechain nodes.
+    Bootstrap a network of sidechain nodes.
     
     Parameters:
-     - network: a map of:
-        - key: the mainchain node
-        - value: an map of information for each sidechain:
-            - key:
-            - value: an array of:
-                - number_of_accounts: the numbers of accounts to be created
-                - withdrawal_epoch_length
-                - websocket_conf: a websocket configuration. An array of:
-                    - address
-                    - connectionTimeout
-                    - reconnectionDelay
-                    - reconnectionMaxAttempts
+     - network: an instance of SCNetworkConfiguration (see sc_boostrap_info.py)
                     
     Example: 2 mainchain nodes and 3 sidechain nodes (with default websocket configuration) bootstrapped, respectively, from mainchain node first, first, and third.
+    The JSON representation is only for documentation.
     {
-        mc_node_1: {
-            1: [3, 1000, [ws://localhost:8888, 100, 1, 1]]
-            2: [3, 1000, [ws://localhost:8888, 100, 1, 1]]
-        }
-        mc_node_2: {
-            1: [3, 1000, [ws://localhost:8888, 100, 1, 1]]
-        }
+        network: [
+            sidechain_1_configuration: {
+                "sc_creation_info":{
+                "sc_id": "id_1"
```

Comment:
1. why `sc_id` is different for different sc nodes? 
2. SC creation info should be at least moved from `sidechain_1_configuration`, to different object and passed as a second argument to `bootstrap_sidechain_nodes`. As i understood, that is what Alberto suggested.

Reply:
Done